### ISSUE TITLE
Cyrillic glyphs?

### ISSUE THREAD

**Issue by marxo** [RedHatBrand/Overpass#14]
Are there any plans to roll out Cyrillic support any time soon? The font is amazing, I'd love to see Cyrillic in it too.

Thanks.

**Comment by inferno986return** [comment]
Highway Gothic is an interesting design and Overpass catches it well. I'd like to see Cyrillic support added too. :-)

**Comment by marxo** [comment]
![2015-12-21 18-14-07](https://cloud.githubusercontent.com/assets/431616/11936543/e7dd0cf4-a80e-11e5-835c-2375b506b741.png)
I was hoping I wouldn't have to do this guys. Can you share some hinting info with me or a script that you generate styles and weights with?

**Comment by marxo** [comment]
![2015-12-26 02-07-45](https://cloud.githubusercontent.com/assets/431616/12004768/025284e6-ab76-11e5-81da-b92d8827ae1a.png)

Finished the Cyrillic glyphs and some basic kerning done! Anyone from the Overpass team willing to connect and help out?

**Comment by inferno986return** [comment]
These are beautiful and seem to fit in nicely with the Highway Gothic style. I hope we get some official feedback soon. :-)

**Comment by andyfitz** [comment]
Hi there,  Cryllic isn't on the immediate roadmap but I have no issue with developing the glyphs in a branch until they are needed in a release.  For now the focus is on expanding the family to heavy and mono variants.

**Comment by naorunaoru** [comment]
Lowercase б and ж are weird, especially б. Others are quite good though!

**Comment by marxo** [comment]
@naorunaoru You are quite right, it felt wrong to me too, so redesigned both of those a while back.
![new glyphs](https://cloud.githubusercontent.com/assets/431616/12640665/6771360c-c5ab-11e5-94f6-c4803f57927b.png)

BTW The whole Cyrillic set is finished, and I'll probably release it soon. I could use help with kerning and scripting, if there's anyone that wants to help feel free to contact me.

**Comment by Jpsy** [comment]
Hi Марко, is there a way you could provide us with your unofficial version of the Overpass including Cyrillic letters? That would be great!

**Comment by andyfitz** [comment]
I'm closing this issue since it didnt make the 3.0 release.

Would love to see Cryllic in Overpass eventually and this may be something down the road for the project.

**Comment by zoran-v-petrovic** [comment]
Lowercase б was correct Serbian glyph, now is Russiаn glyph.
It is well know that there are few Serbian Cyrillic glyphs, usually available with OpenType locl feature.
My post on https://github.com/adobe-fonts/source-sans-pro/issues/38
I hope that You (RedHat or Марко Кажић as a contributor) will include Cyrillic with OpenType LOCL  feature for these Serbian Glyphs. Small б in regular, all other small letter бгдпт in Italic.
This font is sans serif not serif , and Марко Кажић design for letters  г, д, п, т is something between Serbian and Russian glyphs known mostly in serif fonts, letters are more universal, so font can be made without alternate glyphs. For me as a Serbian this Cyrillic glyphs (contribution made by Марко Кажић) is more Serbian than Russian by design.
FontForge as a tool for Cyrillic font design or some commercial product ?
Best Regards.

![2016-12-17 14_40_30-](https://cloud.githubusercontent.com/assets/6688468/21287230/f9c341fc-c466-11e6-8abb-7e4bfcfad846.jpg)

**Comment by davelab6** [comment]
I'd like to request opening this to track it, it's possible that I can bring some Cyrillic designers to help this effort :)

**Comment by zoran-v-petrovic** [comment]
Thanks Dave !

**Comment by StefanPeev** [comment]
@marxo @inferno986return @andyfitz @naorunaoru @Jpsy @davelab6 If Cyrillic will be on the roadmap of this project I'm ready to participate in it offering both traditional Cyrillic, Bulgarian Cyrillic, Serbian and Macedonian Cyrillic Script. So, please. let me know where is the Cyrillic on the roadmap of the project.

**Comment by andyfitz** [comment]
@davelab6   very very late to reopen this,  but it's open again.  
Let's make this a thing

**Comment by davelab6** [comment]
Great! When this gets going, happy to share what we learned in our recent Cyrillic projects - https://design.google/library/scripting-cyrillic/

**Comment by StefanPeev** [comment]
I hope @marxo will share his Cyrillic project. I'm ready to help after that - if it is needed.

**Comment by marxo** [comment]
@StefanPeev @andyfitz I'd love to help. How do I contribute my glyphs to this repo?

**Comment by davelab6** [comment]
@marco first please fork this repo and then checking your work and share the link here. Then we can take a look :)

**Comment by PaulMaly** [comment]
@marxo Is this cyrillic glyphs available somewhere?

**Comment by den32ds** [comment]
@marxo what`s with Cyrillic?

**Comment by sergeevabc** [comment]
No soul shall pass the gates of light,
Where endless summers bide
Until Cyrillic is complete,
And not a glyph denied.
